scripts/extractions/patch_extractions/extract_worldcereal.py

```diff
@@ -0,0 +1,398 @@
+    ds["elevation"] = da
+
+    # Update attributes
+    pipeline_log.info("Updating attributes")
+    ds = ds.assign_attrs(new_attributes)
+
+    # Save to temporary file
+    tempfile = f"./{Path(item_asset_path).name}"
+    pipeline_log.info("Saving to new temporary file")
+    ds.to_netcdf(tempfile)
+
+    # Move to output
+    pipeline_log.info("Moving to original file")
+    shutil.move(tempfile, item_asset_path)
+
+
+def post_job_action_worldcereal(
+    job_items: List[pystac.Item],
+    row: pd.Series,
+    extract_value: int,
+    description: str,
+    title: str,
+    spatial_resolution: str,
+) -> list:
+    """From the job items, extract the metadata and save it in a netcdf file."""
+    base_gpd = gpd.GeoDataFrame.from_features(json.loads(row.geometry)).set_crs(
+        epsg=4326
+    )
+    if len(base_gpd[base_gpd.extract == extract_value]) != len(job_items):
+        pipeline_log.warning(
+            "Different amount of geometries in the job output items and the "
+            "input geometry. Job items #: %s, Input geometries #: %s",
+            len(job_items),
+            len(base_gpd[base_gpd.extract == extract_value]),
+        )
+
+    extracted_gpd = base_gpd[base_gpd.extract == extract_value].reset_index(drop=True)
+    # In this case we want to burn the metadata in a new file in the same folder as the S2 product
+    for item in job_items:
+        item_id = item.id.replace(".nc", "").replace("openEO_", "")
+        sample_id_column_name = "sample_id"
+
+        geometry_information = extracted_gpd.loc[
+            extracted_gpd[sample_id_column_name] == item_id
+        ].squeeze()
+
+        if len(geometry_information) == 0:
+            pipeline_log.warning(
+                "No geometry found for the sample_id %s in the input geometry.",
+                item_id,
+            )
+            continue
+
+        sample_id = geometry_information[sample_id_column_name]
+        valid_time = geometry_information.valid_time
+        s2_tile = row.s2_tile
+
+        item_asset_path = Path(list(item.assets.values())[0].href)
+
+        # Define some metadata to the result_df netcdf file
+        new_attributes = {
+            "start_date": row.start_date,
+            "end_date": row.end_date,
+            "valid_time": valid_time,
+            "GFMAP_version": version("openeo_gfmap"),
+            "creation_date": datetime.now().strftime("%Y-%m-%d %H:%M:%S"),
+            "description": description,
+            "title": title,
+            "sample_id": sample_id,
+            "spatial_resolution": spatial_resolution,
+            "s2_tile": s2_tile,
+            "_FillValue": 65535,  # No data value for uint16
+        }
+
+        # Postprocess file
+        postprocess_extracted_file(item_asset_path, new_attributes)
+
+    return job_items
+
+
+def generate_output_path_worldcereal(
+    root_folder: Path, geometry_index: int, row: pd.Series, s2_grid: gpd.GeoDataFrame
+):
```

Comment:
To return the output type hint

Reply:
@GriffinBabe returns nothing because eventually replaces the original file. Type hint should be -> None then?